pkg/cloudprovider/providers/openstack/openstack.go

```diff
@@ -444,6 +445,65 @@ func getAddressByName(client *gophercloud.ServiceClient, name types.NodeName) (s
 	return addrs[0].Address, nil
 }
 
+// getInterfacesByName returns the node interfaces of the specified instance.
+func getInterfacesByName(client *gophercloud.ServiceClient, name types.NodeName) ([]attachinterfaces.Interface, error) {
+	var interfaces []attachinterfaces.Interface
+
+	// get server
+	srv, err := getServerByName(client, name)
+	if err != nil {
+		if err == ErrNotFound {
+			return interfaces, cloudprovider.InstanceNotFound
```

Comment:
I know such type check happened at somewhere else, but wonder why do we have to define `ErrNotFound` rather than use `cloudprovider.InstanceNotFound1 directly?